src/effects/VST3/VST3Wrapper.cpp

```diff
@@ -312,6 +312,11 @@ VST3Wrapper::~VST3Wrapper()
       mEffectComponent->terminate();
 }
 
+bool VST3Wrapper::IsActive() const noexcept
+{
+   return mActive;
```

Comment:
I saw this new function and thought, is it safe to use only on the main thread?  Do changes of `mActive` happen only in the main thread?  Is `mActive` privately used only on the main thread?

Yes, yes, and yes, which is good.  So please just add a comment in the header file that it's for main thread usage only.

Reply:
>Yes, yes, and yes, which is good. So please just add a comment in the header file that it's for main thread usage only.

The only exception in the interface of this class is a Process function, otherwise it's not thread-safe

Reply:
> > Yes, yes, and yes, which is good. So please just add a comment in the header file that it's for main thread usage only.
> 
> The only exception in the interface of this class is a Process function, otherwise it's not thread-safe

I don't understand that answer.  There is no use of `mActive` but on the main thread.  Correct?

Anyway this unresolved point is only about a comment.